docker-compose.yml

```diff
@@ -44,11 +47,75 @@ services:
         condition: service_healthy
     networks:
       - airbnb_network
+
+  listings:
```

Comment:
I'm just trying to understand what we're using this for? Just not familiar with the approach myself. Mostly how we're using `policies`, `listings` and `celery` directories and docker containers.

Reply:
The `policies` and `listings` containers each encapsulate the code for their respective modules. For example, if you want to manage, run or modify the harvester, you can focus solely on the `listings` module without needing to worry about the `policies` module. 

The `celery` container handles task queuing. In our case, the actual scraping process is performed by Celery. When the harvester endpoint is hit, a task is queued using Redis, and then Celery picks up that task and executes it. This separation "should" allow us to keep the responsibilities of scraping, listings, and policies modular and maintainable.